gapis/api/vulkan/links.go

```diff
@@ -22,6 +22,8 @@ import (
 	"github.com/google/gapid/gapis/service/path"
 )
 
+// When updating this file, you probably also want to update labels.go.
```

Comment:
The file is pretty large and it is easy to miss this comment. I don't fully understand what either links.go or labels.go does and how they relate exactly. Do you think there is a way to get the code of these closer together to make it obvious that these need to be updated/implemented together?

Also, the comment is relatively vague. Without a good understanding of what labels/links are (and there are no file comments), I cannot really figure out what kind of changes would require an update to the other file.

Maybe a file comment to explain what Links/Labels are, what the expectations/contract is would help, too, in addition to explaining how they are related.

Reply:
They go together, because they follow the same pattern - they add functions to the generated types to make them implement certain interfaces. This is Go's implicit interface stuff, which takes some getting used to.

However, I do like the idea of getting into habits of having file level comments to describe what functionality the file contains. If it becomes too hard to make that documentation specific, it probably means the file should be split. So, I've added a comment to `labels.go` and `links.go`.

Reply:
Looks good! That makes it a lot clearer what the files are about and why they may need to updated together.